lua/graph.lua

```diff
@@ -1305,6 +1315,40 @@ function filter_tags_generic(kv)
     kv["bus_backward"] = "true"
   end
 
+  --let all the :forward overrides through
```

Comment:
this is the new code for `:forward` and `:backward` it is certainly redundant for somethings but whatever, i liked have it all in a single spot. i know it will be a terrible chore but the sooner we port this thing to c++ and just drop lua completely, the better